doc/main_page.md

```diff
@@ -11,6 +11,11 @@ In contrast to the Tutorials (which are expected to be useful for all developers
 or specific guides.
 If you have previous experience with SeqAn2 or SeqAn1, have a look at \ref howto_porting.
 
+\ref cookbook contains a listing of code snippets, or *recipes* that might prove helpful once you
+have finished the Tutorials and you are implementing your own code.
```

Comment:
Maybe link the Tutorial here?

Reply:
Good suggestion, unfortunately the tutorial as a whole does not have an index page to reference to. You can only link to the respective chapters of the tutorial. I changed the uppercase to something I feel works better without a hyperlink.